rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/config/ConnectConfig.java

```diff
@@ -111,6 +121,46 @@ public void setNamesrvAddr(String namesrvAddr) {
         this.namesrvAddr = namesrvAddr;
     }
 
+    public int getIsLeader() {
+        return isLeader;
+    }
+
+    public void setIsLeader(int isLeader) {
```

Comment:
Instead of using isLeader, perhaps using a single variable workerRole (or any name you prefer) might be better, and role == 1 means leader, role == 0 means slave, and define these mapping using `public static final int `

Reply:
Updated by using a single variable workerRole instead